components/match2/wikis/overwatch/match_group_input_custom.lua

```diff
@@ -0,0 +1,542 @@
+	match.liquipediatier = Logic.emptyOr(match.liquipediatier, Variables.varDefault('tournament_tier'))
+	match.liquipediatiertype = Logic.emptyOr(match.liquipediatiertype, Variables.varDefault('tournament_tier_type'))
+	match.publishertier = Logic.emptyOr(match.publishertier, Variables.varDefault('tournament_publishertier'))
+	return match
+end
+
+function matchFunctions.getVodStuff(match)
+	match.stream = match.stream or {}
+	match.stream = {
+		stream = Logic.emptyOr(match.stream.stream, Variables.varDefault('stream')),
+		twitch = Logic.emptyOr(match.stream.twitch or match.twitch, Variables.varDefault('twitch')),
+		twitch2 = Logic.emptyOr(match.stream.twitch2 or match.twitch2, Variables.varDefault('twitch2')),
+		afreeca = Logic.emptyOr(match.stream.afreeca or match.afreeca, Variables.varDefault('afreeca')),
+		afreecatv = Logic.emptyOr(match.stream.afreecatv or match.afreecatv, Variables.varDefault('afreecatv')),
+		dailymotion = Logic.emptyOr(match.stream.dailymotion or match.dailymotion, Variables.varDefault('dailymotion')),
+		douyu = Logic.emptyOr(match.stream.douyu or match.douyu, Variables.varDefault('douyu')),
+		smashcast = Logic.emptyOr(match.stream.smashcast or match.smashcast, Variables.varDefault('smashcast')),
+		youtube = Logic.emptyOr(match.stream.youtube or match.youtube, Variables.varDefault('youtube')),
+		facebook = Logic.emptyOr(match.stream.facebook or match.facebook, Variables.varDefault('facebook')),
+	}
+	match.vod = Logic.emptyOr(match.vod, Variables.varDefault('vod'))
+
+	match.lrthread = Logic.emptyOr(match.lrthread, Variables.varDefault('lrthread'))
+
+	match.links = {}
+	local links = match.links
+	if match.preview then links.preview = match.preview end
+	if match.esl then links.esl = 'https://play.eslgaming.com/match/' .. match.esl end
+	if match.owl then links.owl = 'https://overwatchleague.com/en-us/match/' .. match.owl end
+	if match.owc then links.owc = 'https://www.overwatchcontenders.com/match/details/' .. match.owc end
+	if match.jcg then links.jcg = 'http://ow.j-cg.com/compe/view/match/' .. match.jcg end
+	if match.pllg then links.pllg = 'http://peliliiga.fi/' .. match.pllg end
+	if match.oceow then links.oceow = 'http://bmb.oceoverwatch.com/event/' .. match.oceow end
+	if match.tespa then links.tespa = 'https://compete.tespa.org/tournament/' .. match.tespa end
+	if match.overgg then links.overgg = 'http://www.over.gg/' .. match.overgg end
+	if match.pf then links.pf = 'http://www.plusforward.net/overwatch/post/' .. match.pf end
+	if match.wl then links.wl = 'https://www.winstonslab.com/matches/match.php?id=' .. match.wl end
+	if match.stats then links.stats = match.stats end
+
+	-- apply vodgames
+	for index = 1, _MAX_NUM_VODGAMES do
+		local vodgame = match['vodgame' .. index]
+		if not Logic.isEmpty(vodgame) then
+			local map = match['map' .. index] or {}
+			map.vod = map.vod or vodgame
+			match['map' .. index] = map
+		end
+	end
```

Comment:
This can probably be dropped
allows for 
```{{Match|
  |vod = foo
  |vodgame1=abc
  |map1={{Map|....}}
}}```
to be an alias for
```{{Match|
   |vod=foo
   |map1={{Map|vod=abc|...}}
}}